notebooks/book2/17/gprDemoChangeHparams.ipynb

```diff
@@ -2,14 +2,152 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    euc_mat = eucledian_metric((x / len_scale), (x / len_scale))
</pre><p>Perhaps we need to give meaningful names to variables and functions. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>